solidity/contracts/BancorNetwork.sol

```diff
@@ -64,6 +64,13 @@ contract BancorNetwork is IBancorNetwork, TokenHolder, ContractRegistryClient, F
       * @param _registry    address of a contract registry contract
     */
     constructor(IContractRegistry _registry) ContractRegistryClient(_registry) public {
+        etherTokens[address(0)] = true;

```

Comment:
@yudilevi : I initially indeed removed that function, though no need to add the EtherToken address hard-coded as far as I understand. In any case, I then decided to leave it for now. Can't remember why, it might have to do with breaking some of our existing tests. I'll look into it...

Reply:
if we remove that function, we'll need to hardcode the ether token address, I don't see a workaround, but that's fine. We can just define both ether token and address(0) in the ctor.

Reply:
@yudilevi : So this one is done?

Reply:
yeah I think we're good here for now